### PR TITLE
consolidated saas.amazonaws.com template

### PR DESCRIPTION
**Pull request by AbhinavTata**

### CODE REVIEW

File: templates/saas.amazonaws.com.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
+identities:
+- node:
+    name: amazonlinux.us-west-2
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml]: .* the us-west-2

File: templates/saas.amazonaws.com.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    dst_port: 443
+  pushdown: []
+- node:
+    name: email
+    dns_pattern: .*email.*.amazonaws.com
+    dst_port: 443
+  pushdown: []
+- node:
+    name: es
+    dns_pattern: .*es.*.amazonaws.com
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml]: Seems too generic. Can backfire. Remove till we see need

**Review by araalinetworks — CHANGES_REQUESTED** [review]

File: templates/saas.amazonaws.com.yaml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 identities:
+- node:
+    name: amazonlinux..*
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml]: name should not have .*, only _ preferably (not even -)

File: templates/saas.amazonaws.com.yaml [hunk 4]
```diff
@@ -104,16 +174,26 @@ authorizations:
...
+  - pinpoint
+  - pricing
+  - rds
+  - route53
   - s3
   - sns
+  - sqs
   - ssm
   - sts
+  - amazonlinux..*
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml]: bad name

File: templates/saas.amazonaws.com.yaml [hunk 5]
```diff
@@ -63,17 +88,36 @@ identities:
...
+    name: route53
+    dns_pattern: .*route53.*.amazonaws.com
+    dst_port: 443
+  pushdown: []
 - node:
     name: s3
     dns_pattern: .*s3.*.amazonaws.com
     dst_port: 443
-  pushdown:
-  - dns_pattern
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml]: did you intend to remove the pushdown?

File: templates/saas.amazonaws.com.yaml [hunk 6]
```diff
@@ -83,8 +127,12 @@ identities:
     name: sns
     dns_pattern: .*sns.*.amazonaws.com
     dst_port: 443
-  pushdown:
-  - dns_pattern
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml:87]: same here. Do we need to add pushdowns for all of these?

File: templates/saas.amazonaws.com.yaml [hunk 7]
```diff
@@ -103,15 +151,24 @@ identities:
...
   out:
+  - amazonlinux
   - cache
+  - cognito-identity
+  - connect
   - ec2
   - ec2messages
+  - efs
   - eks
-  - eks
```
Comment (araalinetworks) [inline templates/saas.amazonaws.com.yaml:110]: it was there twice?

File: templates/saas.amazonaws.com.yaml (hunk 5 above)
Comment (AbhinavTata) [reply templates/saas.amazonaws.com.yaml]: yes, it wasn't specific enough (didn't have dns pattern listed)

File: templates/saas.amazonaws.com.yaml (hunk 6 above)
Comment (AbhinavTata) [reply templates/saas.amazonaws.com.yaml:87]: i was thinking that, what are your thoughts

File: templates/saas.amazonaws.com.yaml (hunk 7 above)
Comment (AbhinavTata) [reply templates/saas.amazonaws.com.yaml:110]: yes

**Review by araalinetworks — APPROVED** [review]

**Pull request merged**